soundata/annotations.py

```diff
@@ -53,259 +53,38 @@ def __init__(self, intervals, labels, confidence=None) -> None:
         self.confidence = confidence
 
 
-class BeatData(Annotation):
-    """BeatData class
+class Multi_Annotator(Annotation):
```

Comment:
How about `MultiAnnotation` instead? There may be multiple annotations from the same annotator, so I feel like `MultiAnnotation` is a little broader. Also I suggest we remove the "_" for the class name

Reply:
Yeah, let's drop the `_`. About the `MultiAnnotation`, I worry that could be interpreted as the clip having multiple annotations (e.g. tags, events), and that is independent of this class. What do you think?

Reply:
That's a good point, it could be confused with multi-label. OK, let's go with `MultiAnnotator`